gmso/formats/lammpsdata.py

```diff
@@ -38,55 +37,37 @@ def write_lammpsdata(topology, filename, atom_style='full'):
     if atom_style not in ['atomic', 'charge', 'molecular', 'full']:
         raise ValueError('Atom style "{}" is invalid or is not currently supported'.format(atom_style))
 
-    xyz = list()
-    types = list()
-    for site in topology.sites:
-        xyz.append([site.position[0],site.position[1],site.position[2]])
-        types.append(site.atom_type.name)
+    # TODO: Support various unit styles
 
     forcefield = True
+    # Not sure if this is the best way to check if a FF exists
     if topology.sites[0].atom_type.name in ['', None]:
         forcefield = False
 
     box = topology.box
 
-    unique_types = list(set(types))
-    unique_types.sort(key=natural_sort)
-
     # TODO: charges
-    # TODO: bonds
-    # TODO: Angles
-    # TODO: Dihedrals
-
-    # TODO: Figure out handling bond, angle, and dihedral indices
-
-    # placeholder; change later
-    bonds = 0
-    angles = 0
-    dihedrals = 0
 
     with open(filename, 'w') as data:
         data.write('{} written by topology at {}\n\n'.format(
             topology.name if topology.name is not None else '',
             str(datetime.datetime.now())))
         data.write('{:d} atoms\n'.format(len(topology.sites)))
         if atom_style in ['full', 'molecular']:
-            if bonds != 0:
-                data.write('{:} bonds\n'.format(len(bonds)))
+            if len(topology.bonds) != 0:
+                data.write('{:d} bonds\n'.format(len(topology.bonds)))
```

Comment:
So ive not used/encountered  [the format string syntax much.](https://docs.python.org/3.6/library/string.html#format-string-syntax)

But for those who were curious
The colon is used when replacing something in a format string, the colon precedes the format specification for that replacement. See here: [https://docs.python.org/3.6/library/string.html#format-specification-mini-language](https://docs.python.org/3.6/library/string.html#format-specification-mini-language) for more info on the possible specifications that can be used. In the case above ``d`` represents
>

 \'d\' | Decimal Integer. Outputs the number in base 10.
-- | --

Just put here since I went on a bit of a journey learning about this.

Reply:
I was mostly confused about it when it was just `{:}`, since I don't think that does anything

```python3
>>> "x {} z".format('y')
'x y z'
>>> "x {:} z".format('y')
'x y z'
>>> "x {:s} z".format('y')
'x y z'